### DISCUSSION TITLE
Where are the html or svelte pages in docker image?

### DISCUSSION THREAD

**Discussion by MetaLab-UAutonoma** [open-webui/open-webui#3506] [Q&A] [answered]
We have been looking for them because we made a guest user and wanted to add a hard coded button "sign in as guest".
We can't find the page to add this button.
Is it removed or compiled? anyone knows?

**Comment by justinh-rahb** [comment]
It's compiled from the `/src/` dir of the repo when building a docker image, or running `npm run build`.

**Reply by MetaLab-UAutonoma** [reply, marked as answer]
Soo, the only way to add a button is to git clone and build by ourselves?
